com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1182,45 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, application_name,  parameters=[], configuration={}, tracing='info' ):
```

Comment:
Having mutable lists/dicts as the defaults is the incorrect approach for python, as the same object is used for all calls. Instead the default should be `None`.

Reply:
Agreed -- I've changed the method accordingly.